src/components/Card.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import '../styles/Card.css'
+
+const Card = ({ id, poster, original_title, release_date }) => {
+  if (!id) {
+    return <p>Carregando ...</p>
+  }
```

Comment:
Não precisa do fallback aqui, é melhor fazer na lista principal.

Imagina que se não for assim, alguns cards vão ficar carregando e outros aparecerem